### PR TITLE
New CUT&RUN tutorial with Workflow for Training and Workflow for a bigger data analysis 

### PR DESCRIPTION
**Pull request by heylf**
Dear @shiltemann @bgruening @lldelisle,
I need some revision for the CUT&RUN training material. 

- [ ] Test data is ready (two replicates paired-end, down sampled and a ChIP-seq peak file, total 5 files a 30-40 MB). If the training material is ok, then I will upload it to zenodo. Training workflow can be executed within 30 minutes. Sometimes MEME-ChIP can be slow. 
- [ ] Please check if something is missing. 
- [ ] I wanted to include a secondary motif detection. The perfect tool would be [Spamo](https://meme-suite.org/meme/doc/spamo.html) (included in MEME-ChIP, but it is currently deactivated). Is it possible to activate this easily in the MEME-ChIP tool wrapper?

### CODE REVIEW

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (hexylena) [inline topics/epigenetics/tutorials/cut_and_run/tutorial.md:33]:
```suggestion
{CUT&RUN} {% cite Skene2017 %} became the new and advanced method to analyse DNA-associated proteins. CUT&RUN uses an antibody just as [ChIP-Seq](https://en.wikipedia.org/wiki/ChIP_sequencing) to select the protein of interest (POI). The big difference, CUT&RUN couples the antibody with a protein A-micrococcal nuclease (pA-MNase), which you can see in **Figure 1**. The [enzyme](https://en.wikipedia.org/wiki/Micrococcal_nuclease) is an endo-exonuclease that cleaves and shortens the bound DNA of the selected POI *in-situ*. CUT&RUN allows to fragment the DNA in intact cells and thus allows to study protein-DNA interactions in a more natural state. The added pA-MNase thus creates shorted fragments that lead to a higher resolution for the mapping in comparison to your standard ChIP-Seq protocol. CUT&RUN follows four fundamental steps: (1) fixate and make the nuceli or cells permeable, (2) add selective antibody of the POI, (3) add and activate pA-MNas, (4) release DNA complex and collect the DNA from the supernatant. Afterwards, the DNA can be PCR amplified and prepared for sequencing.
```

maybe a good place to use abbreviations instead? It'll render a bit more nicely for screenreaders. https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#abbreviations

just add

```
abbreviations:
  CUT&RUN: Cleavage Under Targets and Release Using Nuclease
```

to the metadata of this tutorial

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (hexylena) [inline topics/epigenetics/tutorials/cut_and_run/tutorial.md:38]:
```suggestion

```

no toc should be avoided completely

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (hexylena) [inline topics/epigenetics/tutorials/cut_and_run/tutorial.md:198]: you need to have `> ` before every line for the box to work

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (heylf) [reply topics/epigenetics/tutorials/cut_and_run/tutorial.md:33]: Awesome. That totally makes sense.

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (hexylena) [reply topics/epigenetics/tutorials/cut_and_run/tutorial.md]: ahh is this branch from a recent version of the gtn? I think that was merged a few weeks ago already but if yours is started earlier.. you could rebase it if you want to use it, otherwise don't worry about it!

Comment (shiltemann) [reply topics/epigenetics/tutorials/cut_and_run/tutorial.md]: we should update planemo to make this the default @hexylena

**Comment by heylf** [comment]
Forgot to ask is there a snippet to create a dataset collection @hexylena

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (lldelisle) [reply topics/epigenetics/tutorials/cut_and_run/tutorial.md:38]: If you have questions or comment on my suggestion, do not hesitate.

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (lldelisle) [inline topics/epigenetics/tutorials/cut_and_run/tutorial.md:40]:
```suggestion
In this tutorial, we will use data from the study of {% cite Zhu2019 %}. The article introduces a CUT&RUN pipeline that we will **not** completely follow. It is important to note at this point that a CUT&RUN data analysis is more similar to an ATAC-Seq experiment than a standard ChIP-Seq. We will analyze the two biological replicates from an CUT&RUN experiment for the aforementioned TF GATA1 in human. We downsampled the data to speed up the run times in this tutorial. Our results will be compared to identified binding sites of GATA1 of a ChIP-Seq experiment {% cite Canver2017 %}.
```

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (lldelisle) [inline topics/epigenetics/tutorials/cut_and_run/tutorial.md:92]: why don't you do a paired collection?

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (lldelisle) [inline topics/epigenetics/tutorials/cut_and_run/tutorial.md:248]:
```suggestion
We apply some filters to the reads after the mapping. CUT&RUN datasets can have many reads that map to the mitchondrial genome because it is nucleosome-free and thus very accessible to the pA(G)-MNase. CUT&RUN inserts adapter more easily in open chromatin regions due to the pA-MNase activity. The mitchondrial genome is uninteresting for CUT&RUN, so we remove these reads. We also remove reads with low mapping quality and reads that are not properly paired.
```

**Review by lldelisle — COMMENTED** [review]
Thanks for proposing this tutorial. I started a review.

File: topics/epigenetics/tutorials/cut_and_run/tutorial.md
Comment (lldelisle) [reply topics/epigenetics/tutorials/cut_and_run/tutorial.md:92]: (maybe you can check the RNA seq tutorial: https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html for an example)

**Comment by lldelisle** [comment]
I think you could add somewhere the link to the protocol: https://dx.doi.org/10.17504/protocols.io.zcpf2vn

**Comment by heylf** [comment]
Thank you @lldelisle for your time :) I have made some changes!

**Comment by lldelisle** [comment]
You are welcome.
Maybe you can add me as 'editing author'. Have you thought about HOMER (findMotifsGenome) for motif detection?

**Comment by lldelisle** [comment]
I don't know. In my lab we use it. It is already in usegalaxy.eu.

**Comment by heylf** [comment]
> I don't know. In my lab we use it. It is already in usegalaxy.eu.

Mmm. I have quite good experience with MEME-ChIP. Maybe good to compare the two tools at some point. Thank you lucille for the recommendation.

File: topics/epigenetics/tutorials/cut_and_run/slides.html
Comment (hexylena) [inline topics/epigenetics/tutorials/cut_and_run/slides.html:25]: would you mind captioning these images?

**Comment by lldelisle** [comment]
Hi @hexylena and @heylf,
If I am not wrong. For the moment these slides are just copy paste from ATAC-seq.

**Comment by heylf** [comment]
Yeah sorry I had the slide from the ATAC-Seq tutorial in there.

**Comment by heylf** [comment]
@lldelisle anymore suggestions? Else I would go on an upload the data to zenodo and prepare the tutorial for the final merge.

**Comment by heylf** [comment]
@lldelisle I uploaded the data to zenodo.

**Comment by lldelisle** [comment]
This needs a second review.

**Comment by bgruening** [comment]
Do you think we can get this as draft in?

**Review by shiltemann — DISMISSED** [review]

**Comment by shiltemann** [comment]
Sure, anything can go in as draft as far as I'm concerned (as long as the build test passes)

**Comment by bgruening** [comment]
Not sure why this is failing to build @shiltemann?

**Review by shiltemann — APPROVED** [review]

**Comment by bgruening** [comment]
Upps, ok good to know, will try to remember this.

**Comment by bgruening** [comment]
Thanks!!!

**Pull request merged**
